doc/visual-programming/source/widgets/data/rank.rst

```diff
@@ -32,13 +32,25 @@ with the class.
 1. Select attributes from the data table.
 2. Data table with attributes (rows) and their scores by different
    scoring methods (columns)
-3. If '*Commit on any change*' ticked, the widget automatically
+3. Produce a report. 
+4. If '*Send Automatically*' is ticked, the widget automatically
    communicates changes to other widgets.
 
+Scoring methods
+---------------
+
+1. `Information Gain <https://en.wikipedia.org/wiki/Information_gain_in_decision_trees>`_: in machine learning, it is synonymous with `Kullback-Leibler divergence <https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence>`_ and in the context of decision trees it can be used interchangeably with the term '*mutual information*'
+2. `Gain Ratio <https://en.wikipedia.org/wiki/Information_gain_ratio>`_: a ratio of information gain to the intrinsic information
+3. `Gini <https://en.wikipedia.org/wiki/Gini_coefficient>`_: measures the inequality among values of a frequency distribution 
+4. `ANOVA <https://en.wikipedia.org/wiki/Analysis_of_variance>`_: a collection of statistical models used to analyze the differences among group means and their associated procedures
+5. `Chi2 <https://en.wikipedia.org/wiki/Chi-squared_distribution>`_: the distribution of a sum of the squares of k independent standard normal random variables
+6. `ReliefF <https://en.wikipedia.org/wiki/Relief_(feature_selection)>`_: a feature selection algorithm used in binary classification
+7. FCBF (Fast Correlation Based Filter)
+
```

Comment:
1. `Information Gain`: the expected amount of information (reduction of entropy)
2. `Gain Ratio <https://en.wikipedia.org/wiki/Information_gain_ratio`_: a ratio of the information gain and the attribute's intrinsic information, which reduces the bias towards multivalued features that occurs in information gain
3. `Gini <https://en.wikipedia.org/wiki/Gini_coefficient>`_: the inequality among values of a frequency distribution 
4. `ANOVA https://en.wikipedia.org/wiki/One-way_analysis_of_variance`_: the difference between average vaules of the feature in different classes
5. `Chi2 <https://en.wikipedia.org/wiki/Chi-squared_distribution>`_: dependence between the feature and the class as measure by the chi-square statistice
6. `ReliefF <https://en.wikipedia.org/wiki/Relief_(feature_selection)>`_: the ability of attribute to distinguish between classes on similar data instances
7. `FCBF (Fast Correlation Based Filter) <https://www.aaai.org/Papers/ICML/2003/ICML03-111.pdf>`_: entropy-based measure which also identifies redundancy due to pairwise correlations between features

This is still not enough, neither for a lay person to understand what these scores actually do, nor for one who knows them but would like to know what exactly Orange computes and by which definitions. But let us assume that we compute the formulas that the links point to?